byron-proxy/src/exec/Main.hs

```diff
@@ -282,7 +303,31 @@ main = withCompileInfo $ do
           fsMountPoint = MountPoint dbFilePath
           fs :: HasFS IO HandleIO
           fs = ioHasFS fsMountPoint
-          openDB = Immutable.openDB
+          -- Must give CSL.HeaderHash encoder/decoder in order to open the
+          -- `ImmutableDB`.
```

Comment:
This is why I made `opendB_NoSerialiseInstance`: I didn't want to have to make a new type or define orphans. `HeaderHash` uses the `cardano-sl` `Bi` instance, and does not have `Serialise` instances.